stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -71,6 +77,10 @@ public class MessageListView : ConstraintLayout {
 
     private lateinit var adapter: MessageListItemAdapter
     private lateinit var layoutManager: LinearLayoutManager
+    private lateinit var loadingView: View
```

Comment:
typically there is also a loading state for loading more. IE scrolling up, which tends to have a different loading UI than the first page load...

Reply:
That's right, I'd like to handle "load more" in another PR to merge this one sooner